### ISSUE TITLE
Facebook

### ISSUE THREAD

**Issue by WalrusInAnus** [easylist/easylist#747]
I am seeing ads absolutely everywhere on Facebook since a few days ago :(

**Comment by gnumdk** [comment]
Same here

**Comment by SXRWahrheit** [comment]
Same

**Comment by WalrusInAnus** [comment]
Is anyone going to do anything about this? Hello? This is rather serious problem.

**Comment by mapx-** [comment]
see if helps
`facebook.com#?#div[data-testid="fbfeed_story"]:-abp-has(a[href*="[is_sponsored]"])`

**Comment by WalrusInAnus** [comment]
Bump.

**Comment by mapx-** [comment]
Did you test the filter above ?

**Comment by WalrusInAnus** [comment]
No, I don't know what to do with it, and I'd rather see a fix implemented into the list itself.
It appears noone from the authors care about Facebook anymore - there have been ads for weeks to months and nothing happened.

**Comment by mapx-** [comment]
if you want a fix you should help with testing proposals.
Add it as custom filter in your adblocker and test it.

**Comment by WalrusInAnus** [comment]
Ok, I tried it.
It shows zero hits despite multiple page refreshes.

There is an suggested pages ad at the top right next to the wall, and "people you may know" multiple times between wall items.
Sometimes there is more than one ad on the right column.

**Comment by mapx-** [comment]
the filter above was only for some sponsored posts in the newsfeed

test these filters:
```
facebook.com#?#div[data-testid="fbfeed_story"]:-abp-has(a[href*="[is_sponsored]"])
facebook.com#?#.fbUserStory:-abp-contains(People you)
facebook.com#?#.userContentWrapper:-abp-contains(People you)
facebook.com#?#.ego_section:-abp-contains(People you)
facebook.com#?#.mtm:-abp-contains(Products shown)
```

if you use english, otherwise adapt them to your language
